pallets/randomness/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{mock::*, Event};
+use frame_support::{
+    assert_ok,
+    pallet_prelude::Weight,
+    traits::{OnFinalize, OnIdle, OnInitialize},
+};
+use sp_runtime::testing::H256;
+
+#[test]
+fn set_babe_randomness_is_mandatory() {
+    use frame_support::dispatch::{DispatchClass, GetDispatchInfo};
+
+    let info = crate::Call::<Test>::set_babe_randomness {}.get_dispatch_info();
+    assert_eq!(info.class, DispatchClass::Mandatory);
+}
+
+#[test]
+fn set_babe_randomness_works() {
```

Comment:
It would be nice to have a test in which you have an epoch that passes with new randomness being set to validate passing from one epoch to another(e.g. `babe_randomness_epoch_change`).

Reply:
I was thinking of improving the mocked data getter to have a test like this, I'll work on it 👍🏻